README.md

```diff
@@ -406,6 +406,70 @@ Loans include information from the item, including locations, holdingsRecordId a
 
 Requests include information from the item, including holdingsRecordId and instanceId.
 
+#### Hold shelf clearance report CSV
```

Comment:
Please remove the reference to CSV from here

Reply:
Fixed